docs/_includes/segmented-buttons.html

```diff
@@ -0,0 +1,122 @@
+<div id="segmented-buttons">
+    {% include section-header.html name="segmented-buttons" version=page.versions.segmentedButton %}
+
+    <p>Segmented Buttons are non-form toggle buttons used in certain cases. For detailed requirements and when to use these over tabs, please visit the <a href="https://ebay.gitbook.io/mindpatterns/input/segmented-buttons">eBay MIND Pattern for Segmented Buttons</a>. </p>
+    <p>The selected toggle button is toggled by <span class="highlight">aria-current="true"</span> attribute.</p>
+    <p><strong>NOTE:</strong> JavaScript is required to handle segmented buttons currently selected button.</p>
+    <p></p>
+
+    <h3 id="button-default">Default Segmented buttons</h3>
+    <div class="demo">
+        <div class="demo__inner">
+            <div class="segmented-buttons">
+                <ul class="segmented-buttons__items">
+                    <li class="segmented-buttons__item"><button type="button" class="btn"
+                            aria-current="true">Day</button></li>
+                    <li class="segmented-buttons__item"><button type="button" class="btn">Month</button></li>
+                    <li class="segmented-buttons__item"><button type="button" class="btn">Year</button></li>
+                </ul>
+            </div>
+
+        </div>
+    </div>
+
+    {% highlight html %}
+<div class="segmented-buttons">
+    <ul class="segmented-buttons__items">
+        <li class="segmented-buttons__item"><button type="button" class="btn"
+                aria-current="true">Day</button></li>
+        <li class="segmented-buttons__item"><button type="button" class="btn">Month</button></li>
+        <li class="segmented-buttons__item"><button type="button" class="btn">Year</button></li>
+    </ul>
+</div>
+    {% endhighlight %}
+
+    <h3 id="button-default">Segmented buttons - with icons</h3>
+    <div class="demo">
+        <div class="demo__inner">
+            <div class="segmented-buttons">
+                <ul class="segmented-buttons__items">
+                    <li class="segmented-buttons__item">
+                        <button type="button" aria-current="true" class="btn">
+                        <span class="btn__cell">
+                                <svg aria-hidden="true" class="icon icon--tick-24" focusable="false" height="16" width="16">
+                                    {% include symbol.html name="tick-24" %}
```

Comment:
Minor, but it seems odd to display a tick here (especially when other button gains aria-current state). Can we have an icon that matches the label of desktop?

Reply:
Sure. I didnt find one which is why i didnt do it.